README.md

```diff
@@ -1,2 +1,258 @@
 # serpens
+
 A set of Python utilities, recipes and snippets
+
+- [SQS Utilities](#sqs-utilities)
+- [API Utilities](#api-utilities)
+- [Schema](#schema)
+- [CSV Utils](#csv-utils)
+- [DynamoDB Documents](#dynamodb-documents)
+
+## SQS Utilities
+
+- This utility is a decorator that iterate by each sqs record.
+
+- For each sqs record will be inserted a *record object (from type sqs.Record)* as  argument that will process the sqs messages.
+
+
+```python
+from serpens import sqs
+
+@sqs.handler
+def message_processor(record: sqs.Record):
+    # code to process each sqs message
+    print(record.body)
+```
+
+### Record
+
+- The function that will process each sqs message receive an instance of *sqs.Record* dataclass. This class has the follow structure:
+
+```python
+class Record:
+    message_id: UUID
+    receipt_handle: str
+    body: Union[str, dict]
+    attributes: Attributes
+    message_attributes: dict
+    md5_of_message_attributes: str
+    md5_of_body: str
+    event_source: str
+    event_source_arn: EventSourceArn
+    aws_region: str
+
+class Attributes:
+    approximate_receive_count: int
+    sent_timestamp: datetime
+    sender_id: str
+    approximate_first_receive_timestamp: datetime
+
+class EventSourceArn:
+    raw: str
+    queue_name: str # is a property
+```
+
+## API Utilities
+
+- This utility is a wrapper for simplify the work with lambda handlers. The decorator ```api.handler``` will decorate a function that will process a lambda and this function will receive a ```request``` argument (from type api.Request).
```

Comment:
@igor-rodrigues1 deixei essa aqui passar no primeiro review. Na minha opinião acho que fica melhor assim o início dessa sentença

```
This utility is a wrapper to simplify working with lambda handlers.
```

Reply:
@rodrigosantiag o PR foi atualizado com os ajustes sugeridos.